src/catalogue/items/catalogueItemsTable.component.test.tsx

```diff
@@ -216,20 +208,21 @@ describe('Catalogue Items Table', () => {
 
     await waitFor(() => {
       expect(
-        screen.getByText(
-          'Precision energy meters for accurate measurements. 26'
-        )
-      ).toBeInTheDocument();
+        screen.getByRole('tooltip', {
+          name: 'Catalogue item description: Precision energy meters for accurate measurements. 26',
+          hidden: true,
+        })
+      ).toBeVisible();
     });
 
     await user.unhover(infoIcon);
 
     await waitFor(() => {
       expect(
-        screen.queryByText(
+        screen.queryAllByText(
           'Precision energy meters for accurate measurements. 26'
-        )
-      ).not.toBeInTheDocument();
+        ).length
+      ).toBe(1);
```

Comment:
I don't think this part of the test makes sense without checking that the length is 2 earlier or something as it could also be that it only finds one regardless.

Reply:
@MatteoGuarnaccia5 Sorry I meant it the test for it should be modified rather than removed. Its a good thing to check it hides again. (Also just occurred to me the `infoIcon` should be renamed now as well`)